src/main/resources/view/MainWindow.fxml

```diff
@@ -1,60 +1,153 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can consider refactoring out in the future, but okay for now

Reply:
Hey, what do you mean? hahah :)

Reply:
Big XML files are hard to read i guess, but dont have to break it up for now.